src/components/structures/RoomView.js

```diff
@@ -259,7 +258,7 @@ module.exports = React.createClass({
                     }
                 }).done();
             }
-        } else if (isUserJoined) {
+        } else {
```

Comment:
I don't really get this if statement now: the bit where we stop peeking is now contingent on us not having a room, which is surely wrong.